src/benchmarks/beluga_vs_nav2_multi_dataset/launch/beluga_vs_nav2_multi_dataset.launch

```diff
@@ -1,80 +1,227 @@
 <launch>
-  <arg name="map_filename"/>
+  <arg name="map_filename" />
```

Comment:
@glpuga meta: why adding these spaces? Google yields some XHTML connection to this formatting style that I don't think is relevant here.